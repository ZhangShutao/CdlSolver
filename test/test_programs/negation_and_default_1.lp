p :- not s.
q :- &c{s}.