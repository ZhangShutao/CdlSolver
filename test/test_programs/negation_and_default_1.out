default model: 1
x:p,q; y:p,s,q;
Satisfiable